src/Action/CompleteRefundPaymentAction.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sylius\RefundPlugin\Action;
+
+use Doctrine\Common\Persistence\ObjectRepository;
+use Sylius\Component\Core\Model\OrderInterface;
+use Sylius\Component\Core\Repository\OrderRepositoryInterface;
+use Sylius\RefundPlugin\Entity\RefundPaymentInterface;
+use Sylius\RefundPlugin\StateResolver\RefundPaymentCompletedStateApplierInterface;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpFoundation\Session\Session;
+use Symfony\Component\Routing\RouterInterface;
+
+final class CompleteRefundPaymentAction
+{
+    /** @var Session */
+    private $session;
+
+    /** @var ObjectRepository */
+    private $refundPaymentRepository;
+
+    /** @var RefundPaymentCompletedStateApplierInterface */
+    private $refundPaymentCompletedStateApplier;
+
+    /** @var RouterInterface */
+    private $router;
+
+    /** @var OrderRepositoryInterface */
+    private $orderRepository;
+
+    public function __construct(
+        Session $session,
+        ObjectRepository $refundPaymentInterface,
+        RefundPaymentCompletedStateApplierInterface $refundPaymentCompletedStateApplier,
+        RouterInterface $router,
+        OrderRepositoryInterface $orderRepository
```

Comment:
Could you change the order of arguments to `OrderRepositoryInterface $orderRepository` be next to `ObjectRepository $refundPaymentInterface`